examples/Python/PET/acquisition_data.py

```diff
@@ -100,12 +100,16 @@ def main():
     print('norm of acq_data.as_array(): %f' % numpy.linalg.norm(acq_array))
     print('acq_data.norm(): %f' % s)
     print('sqrt(acq_data.dot(acq_data)): %f' % math.sqrt(t))
-    diff = new_acq_data - acq_data
+    diff = new_acq_data
+    diff -= acq_data
```

Comment:
this update looks a non-intuitive way to write the code. why would we do this?

Reply:
just was checking `-=`